### PR TITLE
RC1.1 Mirid updates

### PR DESCRIPTION
**Pull request by adrianstevens**

### CODE REVIEW

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Atmospheric.Bmp085.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ The **BMP085** is a high-precision, low-power barometric pressure sensor. The BM
 ### Code Example
 
 ```csharp
-Bmp085 sensor;
+Bmp085? sensor;
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Atmospheric.Bmp085.md:18]: Are these various sensor classes actually structs under the hood, or is this `?` stuff just around the system for explicitly stating when things could possibly be null?

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md [hunk 2]
```diff
@@ -179,8 +179,6 @@ public override Task Run()
             Console.WriteLine();
         }
     }
-
-    return Task.CompletedTask;
```
Comment (patridge) [inline docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md:183]: I see we're still returning this from `Initialize`. Is there a motivation for when or when not to return a completed task vs. nothing vs. `return base.{whatever}`?

**Review by patridge — APPROVED** [review]
Looks good. Had a couple clarification questions, but nothing blocking.

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Atmospheric.Bmp085.md (hunk 1 above)
Comment (adrianstevens) [reply docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Atmospheric.Bmp085.md:18]: yeah it's really just for null warning cleanup

File: docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md (hunk 2 above)
Comment (adrianstevens) [reply docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md:183]: If you add async to the signature the compiler takes care of it. If you're not awaiting any calls, then there's no point in adding async to the signature ... but then we lose the automagical async/await compiler cleanup so we have to manually return a Task.

Comment (patridge) [reply docfx/api-override/Meadow.Foundation/Meadow.Foundation.Sensors.Camera.Mlx90640.md:183]: That kind of makes sense, but in this case, we don't have `async` or a returned task anymore, do we? And even if we `await` something in our override, wouldn't returning/awaiting the `base` implementation be "better". (Though in this case, I am guessing they are basically null implementations.)

**Pull request merged**